### PR TITLE
Automate "Run validation tests" step

### PR DESCRIPTION
**Pull request by boyuanzz**
Automating [Run validation tests](https://beam.apache.org/contribute/release-guide/#run-validation-tests)
Commit https://github.com/apache/beam/pull/6081/commits/7dc4c624e6a793444ae4356d0458a13b79777703: Automate java quickstart validation tests
Commit https://github.com/apache/beam/pull/6081/commits/7210ead147971388465a2390dbd923bb8963c16f: Automate java mobile game validations
Commit https://github.com/apache/beam/pull/6081/commits/5dba7d085fc29f26779164827906341956c91ec1: Automate python quickstart & userscore & hourlyteamscore
Commit https://github.com/apache/beam/pull/6081/commits/96ef34223a310a2aa24342db350a2b5e7625ac36: Automate python Leaderboard & GameStats

Overall needs more test on this script.
r: @aaltay @pabloem

### CODE REVIEW

File: release/src/main/scripts/run_rc_validation.sh [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+  ls | grep 'apache-beam-${RELEASE}'
+
+  echo "--------------------------Verifying Hashes------------------------------------"
+  sha512sum -c apache-beam-${RELEASE}.zip.sha512
+
+  echo "----------------------------Building Python SDK-------------------------------"
+  sudo apt-get install unzip
+  unzip apache-beam-${RELEASE}.zip
+  cd apache-beam-${RELEASE}
+  python setup.py sdist
```
Comment (aaltay) [inline release/src/main/scripts/run_rc_validation.sh]: Why do we build as part of validation?

**Comment by aaltay** [comment]
Is this ready for review?

**Comment by boyuanzz** [comment]
This PR is ready to review @aaltay

File: release/src/main/scripts/run_rc_validation.sh (hunk 1 above)
Comment (boyuanzz) [reply release/src/main/scripts/run_rc_validation.sh]: Addressed in commit https://github.com/apache/beam/pull/6081/commits/a2303400f6fad8c9cda22395be24697bb4f1efed

**Pull request merged**